api/notd/token_metadata_processor.py

```diff
@@ -302,6 +304,8 @@ async def retrieve_token_metadata(self, registryAddress: str, tokenId: str, coll
             tokenMetadataDict = tokenMetadataDict[0] if len(tokenMetadataDict) > 0 else {}  # type: ignore[assignment]
         if not isinstance(tokenMetadataDict, dict):
             return self.get_default_token_metadata(registryAddress=registryAddress, tokenId=tokenId)
+        if registryAddress == '0x495f947276749Ce646f68AC8c248420045cb7b5e':
+            await self.subCollectionTokenManager.update_sub_collection_token(registryAddress=registryAddress, tokenId=tokenId)
```

Comment:
its possible this call should be moved all the way out to the notdManager, what do you think?